### PR TITLE
#15227 support display name in lineage search node

### LINKED ISSUE #15227: Show Display Name while adding new node in Lineage
image

Show display name if available, for empty display name show name only.

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes #15227

- I worked on support display name in lineage search node

image
image
image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx [hunk 1]
```diff
@@ -604,7 +603,7 @@ export const createNodes = (
       className: '',
       data: {
         node,
-        isRootNode: getDecodedFqn(entityFqn) === node.fullyQualifiedName,
+        isRootNode: entityFqn === node.fullyQualifiedName,
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx:606]: Don't we need to decode the fqn here? can explain why?

File: openmetadata-ui/src/main/resources/ui/src/components/Entity/EntityLineage/NodeSuggestions.component.tsx [hunk 2]
```diff
@@ -139,14 +139,19 @@ const NodeSuggestions: FC<EntitySuggestionProps> = ({
...
                         entity.entityType as string
                       )}
                     </p>
                   )}
-                  <p className="w-max-400 truncate">{entity.name}</p>
+                  <p className="text-xs text-grey-muted w-max-400 truncate line-height-normal">
+                    {entity.name}
+                  </p>
+                  <p className="w-max-400 text-sm font-medium truncate">
+                    {entity.displayName}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Entity/EntityLineage/NodeSuggestions.component.tsx:153]: displayName can be undefined right, so can we use `getEntityName` here to have the fallback?

File: openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx:606]: We don't need to decode here, as we are already passing decodedFqn as an arguments.

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx:606]: Thanks for confirming @Ashish8689

**Pull request merged**